libretroshare/src/retroshare/rsnotify.h

```diff
@@ -221,6 +221,7 @@ class NotifyClient
 	virtual void notifyChatLobbyEvent             (uint64_t           /* lobby id */, uint32_t           /* event type    */ ,const RsGxsId& /* nickname */,const std::string& /* any string */) {}
 	virtual void notifyChatLobbyTimeShift         (int                /* time_shift*/) {}
 	virtual void notifyCustomState                (const std::string& /* peer_id   */, const std::string&               /* status_string */) {}
+  virtual void notifyReputationChange           (RsGxsId id) {}
```

Comment:
Tab indent.
Why adding at this line?
In p3notify.h, it's defined before notifyChatLobbyEvent.
It's already enough a mess. ;)